projects/bp-gallery/cypress/e2e/facetags.cy.ts

```diff
@@ -0,0 +1,105 @@
+import { login, logout } from '../utils/login-utils';
+
+describe('face tagging', () => {
+  before(() => {
+    cy.visit('/');
+    login();
+    cy.visit('/picture/1');
+  });
+
+  after(() => {
+    cy.visit('/');
+    cy.get('.nav-bar').contains('Mehr...').click();
+    cy.get('.MuiPaper-root').contains('Personen').click();
+    cy.contains('.MuiDataGrid-row', 'TestPerson').find('[data-testid="DeleteIcon"]').click();
+    cy.get('.MuiButton-root').contains('Bestätigen').click();
+    cy.contains('.MuiDataGrid-row', 'TestPerson').should('not.exist');
+    cy.contains('.MuiDataGrid-row', 'Personentest').find('[data-testid="DeleteIcon"]').click();
+    cy.get('.MuiButton-root').contains('Bestätigen').click();
+    cy.contains('.MuiDataGrid-row', 'Personentest').should('not.exist');
+    cy.visit('/');
+    logout();
+  });
+
+  it('create face tags', () => {
+    cy.get('.facetag').should('not.exist');
+    cy.contains('Personen markieren').should('not.exist');
```

Comment:
ich fände es in solchen instanzen sinnvoller wenn man hierfür lieber Klassennamen anstatt written text benutzt, weil ansonsten eine Änderung in de.json den ganzen Test kaputt gehen lässt. Müsst ihr aber jetzt nicht umschreiben

Reply:
Das haben wir und auch das alte BP über alle Tests hinweg gemacht. Wäre also ein ziemlicher Aufwand ;)